docs/src/mdx-components.tsx

```diff
@@ -65,7 +65,9 @@ export const hr = () => <Separator spaceAfter="largest" />;
 export const ol = ({ children }: React.OlHTMLAttributes<HTMLOListElement>) => (
   <ol
     css={css`
-      list-style: outside none decimal;
+      list-style-position: outside;
+      list-style-image: none;
+      list-style-type: decimal;
```

Comment:
We ordered the values in the shorthand wrong, according to [`list-style`'s MDN documentation](http://developer.mozilla.org/en-US/docs/Web/CSS/list-style) the correct order is `type | image | position`, which works in Safari.
```suggestion
      list-style: decimal none outside;
```